modin/backends/pandas/query_compiler.py

```diff
@@ -776,12 +776,17 @@ def unique(self):
     dt_days = MapFunction.register(_dt_prop_map("days"))
     dt_microseconds = MapFunction.register(_dt_prop_map("microseconds"))
     dt_nanoseconds = MapFunction.register(_dt_prop_map("nanoseconds"))
-    dt_components = MapFunction.register(_dt_prop_map("components"))
     dt_qyear = MapFunction.register(_dt_prop_map("qyear"))
     dt_start_time = MapFunction.register(_dt_prop_map("start_time"))
     dt_end_time = MapFunction.register(_dt_prop_map("end_time"))
     dt_to_timestamp = MapFunction.register(_dt_func_map("to_timestamp"))
 
+    def dt_components(self):
+        new_modin_frame = self._modin_frame._apply_full_axis(
+            1, lambda df: df.squeeze().dt.components
```

Comment:
We need to use `df.squeeze(axis=1)` here because this won't work for `Series` with one element. Also, there is no scaling here. Are there a another way to implement this?